app/plugins/Wallet/js/buttons.js

```diff
@@ -106,7 +106,7 @@ addResultListener('coin-sent', function(result) {
 // Lock or unlock the wallet
 eID('lock-pod').onclick = function() {
 	var state = eID('lock-status').innerHTML;
-	if (!wallet.unlocked && state === 'Unencrypted') {
+	if (!wallet.encrypted && state === 'Create Wallet') {
 		encrypt();
```

Comment:
The element now contains the text "Create Wallet" instead of "Unencrypted" so that it's more clear that it's a button that the user should click. Unfortunately this makes use of the implicit assumption that `!wallet.encrypted` means the wallet has not been created.

We could probably remove the `state` checks entirely.